requirements/dev

```diff
@@ -1,3 +1,4 @@
+future
```

Comment:
this is in the `base` requirements already, do you think we need it here too?

Reply:
Will remove...missed it there.